src/main/java/ui/MoveMethodPanel.java

```diff
@@ -60,6 +63,7 @@
     private final JButton refreshButton = new JButton();
     private final List<MoveMethodRefactoring> refactorings = new ArrayList<>();
     private JScrollPane scrollPane = new JScrollPane();
+    private final JButton exportButton = new JButton();
```

Comment:
This button should be enabled only if there are results that can be exported. 
See `enableButtonsOnConditions() `and `disableAllButtons()`.